Adds screen reader text to pagination - trello ticket #463

BNewing opened the pull request:
This adds text to the pagination links. I've added text to the previous and next links as well as the numbers.

BNewing committed:
Adds screen reader text to pagination - trello ticket #463

This adds text to the pagination links. I've added text to the previous and next links as well as the numbers.

wilson1000-MoJ commented on web/app/themes/justicejobs/search-apply-page.php:
```diff
@@ -263,16 +263,16 @@ class="search_contain__radio search_contain__radio--map"
+            'prev_text' => '<span class="screen-reader-text"> Search results - previous page</span><span aria-hidden="true">PREV</span>',
+            'next_text' => '<span class="screen-reader-text"> Search results - </span>NEXT <span class="screen-reader-text">page</span>',
+            'before_page_number' => '<span class="screen-reader-text"> Search results - page </span>',
+            'after_page_number' => '<span class="screen-reader-text"> of ' . __( $job_query->max_num_pages ). '</span>'
```
__() is a WP translation function used to display (if present) the text in a different language.
It is best practice to define a text-domain in the themes root style.css file and then pass the text-domain as a string in the second parameter in each function call. This is regarded as good code development as it essentially creates a theme translation layer and will make translation processes easier in future. 
Would you kindly add the text-domain?
Ref: https://developer.wordpress.org/reference/functions/__/

wilson1000-MoJ commented on web/app/themes/justicejobs/search-apply-page.php:
```diff
@@ -388,30 +388,13 @@ class="map"
+            'prev_text' => '<span class="screen-reader-text"> Search results - previous page</span><span aria-hidden="true">PREV</span>',
+            'next_text' => '<span class="screen-reader-text"> Search results - </span>NEXT <span class="screen-reader-text">page</span>',
+            'before_page_number' => '<span class="screen-reader-text"> Search results - page </span>',
+            'after_page_number' => '<span class="screen-reader-text"> of ' . __( $job_query->max_num_pages ). '</span>'
```
As above

wilson1000-MoJ commented on web/app/themes/justicejobs/search.php:
```diff
@@ -582,8 +582,10 @@ class="search_contain__radio search_contain__radio--map"
+                    'prev_text' => '<span class="screen-reader-text"> Search results - previous page</span><span aria-hidden="true">PREV</span>',
+                    'next_text' => '<span class="screen-reader-text"> Search results - </span>NEXT <span class="screen-reader-text">page</span>',
+                    'before_page_number' => '<span class="screen-reader-text"> Search results - page </span>',
+                    'after_page_number' => '<span class="screen-reader-text"> of ' . __( $job_query->max_num_pages ). '</span>'
```
Add text-domain to __()

wilson1000-MoJ commented on web/app/themes/justicejobs/search.php:
```diff
@@ -721,29 +723,15 @@ class="map"
+                    'prev_text' => '<span class="screen-reader-text"> Search results - previous page</span><span aria-hidden="true">PREV</span>',
+                    'next_text' => '<span class="screen-reader-text"> Search results - </span>NEXT <span class="screen-reader-text">page</span>',
+                    'before_page_number' => '<span class="screen-reader-text"> Search results - page </span>',
+                    'after_page_number' => '<span class="screen-reader-text"> of ' . __( $job_query->max_num_pages ). '</span>'
```
Add text-domain to __()

BNewing committed:
Add translation capability to pagination screen reader text

This allows the text available to screen reader users to be translated. I've added in the text domain to the style.css, which the text snippets reference. The structure of the overall sentences can't be changed, because of the way we need to pull in values from the PHP, but from a quick look at Welsh grammar in translations, this should be okay.

Whilst there is some repetition of words, I've kept the strings largely as they are without trying to shorten them. This should help translators understand the overall string, and lower the risk of translations being incorrect (for example if words need a declination, or needs to be in a certain position depending on the context).

BNewing commented:
Thank you for the change requests @wilson1000-MoJ ! I've now added another commit to address this feedback
